include/ginkgo/core/matrix/csr.hpp

```diff
@@ -158,6 +158,22 @@ class Csr : public EnableLinOp<Csr<ValueType, IndexType>>,
 public:
     using EnableLinOp<Csr>::convert_to;
     using EnableLinOp<Csr>::move_to;
+    using ConvertibleTo<Csr<next_precision<ValueType>, IndexType>>::convert_to;
+    using ConvertibleTo<Csr<next_precision<ValueType>, IndexType>>::move_to;
+    using ConvertibleTo<Dense<ValueType>>::convert_to;
+    using ConvertibleTo<Dense<ValueType>>::move_to;
+    using ConvertibleTo<Coo<ValueType, IndexType>>::convert_to;
+    using ConvertibleTo<Coo<ValueType, IndexType>>::move_to;
+    using ConvertibleTo<Ell<ValueType, IndexType>>::convert_to;
+    using ConvertibleTo<Ell<ValueType, IndexType>>::move_to;
+    using ConvertibleTo<Fbcsr<ValueType, IndexType>>::convert_to;
+    using ConvertibleTo<Fbcsr<ValueType, IndexType>>::move_to;
+    using ConvertibleTo<Hybrid<ValueType, IndexType>>::convert_to;
+    using ConvertibleTo<Hybrid<ValueType, IndexType>>::move_to;
+    using ConvertibleTo<Sellp<ValueType, IndexType>>::convert_to;
+    using ConvertibleTo<Sellp<ValueType, IndexType>>::move_to;
+    using ConvertibleTo<SparsityCsr<ValueType, IndexType>>::convert_to;
+    using ConvertibleTo<SparsityCsr<ValueType, IndexType>>::move_to;
```

Comment:
it's necessary because in an overload set, only the last class to overload a function actually has its member functions visible by default. These `using` statements pull in the `pointer_param` overloads, as they are not visible by default.